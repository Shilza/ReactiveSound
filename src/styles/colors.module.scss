
$colors: (
        'primary': #262728,
        'accent': #aaa,
        'secondary': #1d1e1f,
        'background': #262728
)