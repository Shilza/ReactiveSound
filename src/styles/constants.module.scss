$constants: (
        'maxWidth': 1000px
);
