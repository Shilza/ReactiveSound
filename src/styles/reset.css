
ul {
    list-style: none;
    padding: 0;
    margin: 0;
}

a {
    margin: 0;
    padding: 0;
    font-size: 100%;
    vertical-align: baseline;
    background: transparent;
    text-decoration: unset;
    font-weight: normal;
    color: currentColor;
}