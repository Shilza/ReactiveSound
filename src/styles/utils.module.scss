
@import 'colors.module';

@function color($key: 'primary') {
  @return map-get($colors, $key);
}