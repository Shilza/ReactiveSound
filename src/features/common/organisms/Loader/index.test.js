import React from "react";
import {render} from 'react-testing-library';
import {Loader} from "./index";
import styles from './styles.module.scss';

describe('Test Loader', () => {
    const {container} = render(<Loader/>);
    const childNodes = container.firstChild.childNodes;

    it('should render correctly', () => {
        expect(container.firstChild).toMatchSnapshot();
    });

    it('should have three child', () => {
        expect(childNodes.length).toBe(3);
    });

    it('first child should have circle class', () => {
        expect(childNodes[0]).toHaveClass(styles.circle)
    });

    it('second child should have circle2 class', () => {
        expect(childNodes[1]).toHaveClass(styles.circle2)
    });

    it('third child should have circle3 class', () => {
        expect(childNodes[2]).toHaveClass(styles.circle3)
    });
});