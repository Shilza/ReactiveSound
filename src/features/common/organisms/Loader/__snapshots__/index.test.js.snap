// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Loader should render correctly 1`] = `
<div
  class="loadingIndicator"
  data-testid="loader"
>
  <div
    class="circle"
  />
  <div
    class="circle2"
  />
  <div
    class="circle3"
  />
</div>
`;
